apps/aggiemap/src/app/modules/components/trip-planner/trip-planner-options/containers/base/base.component.ts

```diff
@@ -99,7 +103,7 @@ export class TripPlannerOptionsComponent implements OnInit {
     this.plannerService.updateTravelOptions(opt);
   }
 
-  public getOptionvalue(option): Observable<any> {
+  public getOptionvalue(option) {
```

Comment:
This might be more explicit, instead of returning `Observable<unknown>`?

```
  public getOptionvalue<T>(option: string): Observable<T> {
    return this.travelOptions.pipe(pluck(option));
  }
```